services/stats.go

```diff
@@ -199,6 +199,21 @@ func eth1UniqueValidatorsCount() (*uint64, error) {
 }
 
 // GetValidatorChurnLimit returns the rate at which validators can enter or leave the system
```

Comment:
This tool tip contains the wrong function name (`GetValidatorChurnLimit`).
Also, is it correct? Based on the function name I would assume that this is only about the enter rate and not about the leave rate.